code/modules/tram/tram.dm

```diff
@@ -0,0 +1,227 @@
+/obj/tram/ex_act(severity)
+	switch(severity)
+		if(1.0)
+			del(src)
+			return
+		if(2.0)
+			if(prob(50))
+				del(src)
+			return
+		if(3.0)
+			if(prob(25))
+				del(src)
+			return
+
+/obj/tram/blob_act()
+	if(prob(50))
+		del(src)
```

Comment:
we don't have qdel yet.

On Thu, Jun 4, 2015 at 7:49 PM, Thomas Stoeckert <notifications@github.com>
wrote:

> In code/modules/tram/tram.dm
> <https://github.com/UristMcStation/UristMcStation/pull/229#discussion_r31777509>
> :
>
> > +	switch(severity)
> > +		if(1.0)
> > +			del(src)
> > +			return
> > +		if(2.0)
> > +			if(prob(50))
> > +				del(src)
> > +			return
> > +		if(3.0)
> > +			if(prob(25))
> > +				del(src)
> > +			return
> > +
> > +/obj/tram/blob_act()
> > +	if(prob(50))
> > +		del(src)
>
> What are the benefits of qdel over just del?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UristMcStation/UristMcStation/pull/229/files#r31777509>
> .
>



-- 
Graham Lloyd